daemon/container.go

```diff
@@ -705,9 +711,12 @@ func (container *Container) Stop(seconds int) error {
 	// 1. Send a SIGTERM
 	if err := container.KillSig(15); err != nil {
 		log.Infof("Failed to send SIGTERM to the process, force killing")
-		if err := container.KillSig(9); err != nil {
-			return err
+		if err != syscall.ESRCH {
+			if err := container.KillSig(9); err != nil && err != syscall.ESRCH {
```

Comment:
But that would just duplicate the message if the process terminates before `kill 15` with no message in between.